### PR TITLE
Allow easier extension to help command by removing switch case

### PR DESCRIPTION
**Pull request by yixiann**
I realised there was a slightly better way to do this. 

Now we only need to specify what is the help message in the help command and the command word.

No changes need to be made to any switch case block.

### CODE REVIEW

**Review by RichDom2185 — APPROVED** [review]
LGTM, nice, thanks for this! Just one comment.

File: src/test/java/seedu/foodrem/logic/parser/generalcommandparser/HelpCommandParserTest.java [hunk 1]
```diff
@@ -112,101 +109,102 @@ void parse_generalCommands() {
...
         assertEquals(ListTagCommand.MESSAGE_USAGE,
-                getHelp(getCommandWordFromCommandWordString("listtag")));
+                getCommandWordFromCommandWordString("listtag").getHelp());
     }
 
     @Test
     void getHelp_throws_error() {
-        assertThrows(ParseException.class, () -> getHelp(getCommandWordFromCommandWordString("helps")));
+        assertEquals(DEFAULT.getHelp(),
+                getCommandWordFromCommandWordString("helps").getHelp());
```
Comment (RichDom2185) [inline src/test/java/seedu/foodrem/logic/parser/generalcommandparser/HelpCommandParserTest.java:208]: What was this change about?

**Review by RichDom2185 — DISMISSED** [review]
LGTM, nice, thanks for this! Just one comment.

File: src/test/java/seedu/foodrem/logic/parser/generalcommandparser/HelpCommandParserTest.java (hunk 1 above)
Comment (RichDom2185) [reply src/test/java/seedu/foodrem/logic/parser/generalcommandparser/HelpCommandParserTest.java:208]: On a second look, I can see why this change was made. In that case, should we combine this with the `get_help_success` method?

Comment (yixiann) [reply src/test/java/seedu/foodrem/logic/parser/generalcommandparser/HelpCommandParserTest.java]: Ah thank you for pointing this out. I did not realise it when changing.

File: src/test/java/seedu/foodrem/logic/parser/generalcommandparser/HelpCommandParserTest.java [hunk 2]
```diff
@@ -20,22 +37,46 @@
...
+        3) getHelp_TYPEOFCOMMAND
+
+        Where TYPEOFCOMMAND is either generalCommands / itemCommands / tagCommands
+
+        Thank you :D
+        */
+        int numberOfCommands = 19;
+        assertEquals(numberOfCommands, CommandWord.values().length);
+    }
+
```
Comment (RichDom2185) [inline src/test/java/seedu/foodrem/logic/parser/generalcommandparser/HelpCommandParserTest.java:71]: Nice, I like this test!

**Review by RichDom2185 — APPROVED** [review]
LTGM, thanks for this!

**Pull request merged**